pyproject.toml

```diff
@@ -60,6 +60,8 @@ gptqmodel = [
   { index = "pruna_internal", marker = "sys_platform != 'darwin' or platform_machine != 'arm64'"},
   { index = "pypi", marker = "sys_platform == 'darwin' and platform_machine == 'arm64'"},
 ]
+image-reward = { git = "https://github.com/PrunaAI/ImageReward" }
+hpsv2 = { git = "https://github.com/PrunaAI/HPSv2" }
```

Comment:
There were some issues with both repositories.
ImageReward, had pushed their fixes to `main` 
HPSv2 had very strict constraints for protobuf where they set it to lower than 6